src/Microsoft.Health.Dicom.Core/Models/Operations/DurableOrchestrationInstanceStatus.cs

```diff
@@ -0,0 +1,73 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Dicom.Core.Models.Operations.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Core.Models.Operations
+{
+    /// <summary>
+    /// Represents the metadata for an Azure Durable Orchestration instance.
+    /// </summary>
+    internal class DurableOrchestrationInstanceStatus
+    {
+        /// <summary>
+        /// Gets or sets the type of the orchestration as understood by the DICOM server.
+        /// </summary>
+        /// <value>
+        /// The <see cref="OperationType"/> if the name is recognized; otherwise <see cref="OperationType.Unknown"/>.
+        /// </value>
+        [JsonProperty("Name")]
+        [JsonConverter(typeof(OperationTypeConverter))]
```

Comment:
Just a idea, no code change required. 
I'm thinking if we should  simplify conversion logic (`DurableOrchestrationStatus` => `json string` =>`DurableOrchestrationInstanceStatus` =>`OperationStatusResponse`)  by moving most logic into Dicom Function side.
Another though is that I feel DurableOrchestrationInstanceStatus coupled too much with Azure Function, which we may not want.

Reply:
I actually do want some amount of coupling. For example, I would like the `IDicomOperationsClient` to serve as the abstraction that is agnostic of how we're running our long-running operations. So the types it operates upon and returns must be agnostic; thus the `OperationStatusResponse` is independent of the technology. However, the implementation for Azure Durable Functions, the `DicomDurableFunctionsHttpClient`, depends on the types return by Azure Durable Functions and the REST APIs we write. In that way, it is reasonable for this class to internally use durable-function specific types/method/etc.

However, the client must transform the Azure Function-specific payloads into more agnostic ones for general consumption.